modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/legacy/scheduler_event_recorder.js

```diff
@@ -660,6 +661,44 @@ AUI.add(
 					messageNode.innerHTML = messageHTML;
 				},
 
+				getFormattedDate() {
+					const instance = this;
+					const event = instance.get('event') || instance;
+					const endDate = event.get('endDate');
+					const startDate = event.get('startDate');
+
+					const formattedStartDate = event._formatDate(
+						startDate,
+						instance.get('dateFormat')
+					);
+
+					if (event.get('allDay')) {
+						return formattedStartDate;
+					}
+
+					let formattedEndDate = event._formatDate(
+						endDate,
+						instance.get('dateFormat')
+					);
+
+					if (formattedEndDate === formattedStartDate) {
+						formattedEndDate = '';
+					}
+
+					const scheduler = event.get('scheduler');
+					const fmtHourFn = scheduler.get('activeView').get('isoTime')
+						? DateMath.toIsoTimeString
+						: DateMath.toUsTimeString;
+
+					return [
+						formattedStartDate.concat(','),
```

Comment:
Hey @pedro-oliveira446, could you check if in the case of events that end on the same day, we will have an extra unwanted comma here?

Reply:
The comma after the start date is used to separate it from the hour, so it is always necessary.